packages/injective-math/src/utils.rs

```diff
@@ -75,6 +76,34 @@ pub fn div_dec(num: FPDecimal, denom: FPDecimal) -> FPDecimal {
     }
 }
 
+pub fn floor(num: FPDecimal, min_tick: FPDecimal) -> FPDecimal {
+    // min_tick has to be a positive number
+    //FIXME min_tick has to be 1, 0.1,0.01, etc
```

Comment:
it could be 10, 100... also right? then how about instead of `min_tick: FPDecimal` you pass in an `power_scaling: i32` which represents the 10^number ? you can use the `FPDecimal.scaled()` function to calculate the result